docs/recap/bitvm2_fast_recap.tex

```diff
@@ -0,0 +1,229 @@
+% Created 2024-09-23 Mon 13:35
+% Intended LaTeX compiler: pdflatex
+%%% TeX-command-extra-options: "-shell-escape"
+\documentclass[11pt]{article}
+\usepackage[utf8]{inputenc}
+\usepackage[T1]{fontenc}
+\usepackage{graphicx}
+\usepackage{longtable}
+\usepackage{wrapfig}
+\usepackage{rotating}
+\usepackage[normalem]{ulem}
+\usepackage{amsmath}
+\usepackage{amssymb}
+\usepackage{capt-of}
+\usepackage{hyperref}
+\makeatletter
+\newcommand{\citeprocitem}[2]{\hyper@linkstart{cite}{citeproc_bib_item_#1}#2\hyper@linkend}
+\makeatother
+\usepackage[a4paper, total={6in, 8in}]{geometry}
+\author{Kyrylo Baibula}
+\date{\today}
+\title{BitVM fast recap}
+\hypersetup{
+  pdfauthor={Kyrylo Baibula},
+  pdftitle={BitVM fast recap},
+  pdfkeywords={},
+  pdfsubject={},
+  pdfcreator={Emacs 29.4 (Org mode 9.7.11)},
+pdflang={English}}
+\usepackage{biblatex}
+
+\addbibresource{refs.bib}
+
+\begin{document}
+
+\maketitle
+
+\begin{abstract}
+  Assume some user wants to publicly execute a large program on the
+  Bitcoin chain using Bitcoin Script, but its implementation is larger
+  than 4 megabytes (for example, a Zero Knowledge proof); that is why it
```

Comment:
for example, a zero-knowledge verification logic

Reply:
instead of "4 megabytes; that is why..." use "4 megabytes, making it impossible to publish the complex logic on-chain"